### ISSUE TITLE
Is fly.io still free?

### ISSUE THREAD

**Issue by 0x00000000000000000000000** [255kb/stack-on-a-budget#317]
https://github.com/255kb/stack-on-a-budget/blob/master/pages/app-hosting.md#fly

![image](https://github.com/user-attachments/assets/37603f77-2cc6-441e-a402-29992a46c7d9)

I can't find the pricing for `Fly Machines`, https://fly.io/docs/machines/

**Comment by 255kb** [comment]
Indeed, it seems the free hobby plan was [discontinued](https://fly.io/docs/about/pricing/#legacy-hobby-plan). I think we could remove fly.io
